app/models/like.rb

```diff
@@ -0,0 +1,4 @@
+class Like < ApplicationRecord
```

Comment:
Please add commands `rails db:create`, `rails db:migrate`, and `npm install or yarn install` in README for the user to set up the project locally.